emctl/cmd/client/command/meshinstall/base/k8sutils.go

```diff
@@ -570,11 +570,19 @@ func GetMeshControlPlaneEndpoints(client kubernetes.Interface, namespace, resour
 	entrypoints := []string{}
 	for _, n := range nodes.Items {
 		for _, i := range n.Status.Addresses {
+			address := i.Address
+
+			// NOTICE: For situation where the node address is not reachable.
+			if os.Getenv("EMCTL_NODE_ADDRESS") != "" {
+				address = os.Getenv("EMCTL_NODE_ADDRESS")
```

Comment:
To avoid possible time of check to time of use issue:

```suggestion
			if addr := os.Getenv("EMCTL_NODE_ADDRESS"); addr != "" {
				addrress = addr
```

Reply:
Sure, 😂but your typo got me

Reply:
Sorry for that...